### DISCUSSION TITLE
Azure API

### DISCUSSION THREAD

**Discussion by mgajownik** [AzureCommunityPL/group#315] [Pomoc]
Cześć z rana :)

macie jakieś źródło, wiedzę o czasie odpowiedzi Azure API. case jest taki mam Forti on wprowadza zmiany na RT, NSG, vnety itd i pytanie po jakim czasie mamy odpowiedź?

**Comment by kamil-mrzyglod** [comment]
Ale o które dokładnie API Ci chodzi? Azure Resource Manager? Chodzi Ci o czas odpowiedzi na zapytanie typu POST/PATCH czy event z Event Grid?

**Comment by sbugalski** [comment]
Pytasz po jakim czasie Resource Manager zwróci ci 201 / 202?
Czy po jakim czasie zostaną wprowadzone zmiany w asynchronicznych operacjach?

https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/async-operations
